src/Symmetry/Printing.jl

```diff
@@ -192,13 +198,13 @@ end
 
 
 """
-    print_bond_table(cryst::Crystal, max_dist)
+    print_symmetry_table(cryst::Crystal, max_dist)
 
 Pretty-prints a table of bonds, one for each symmetry equivalence class, up to a
 maximum bond length of `max_dist`. Equivalent to calling `print_bond(cryst, b)`
 for every bond `b` in `reference_bonds(cryst, max_dist)`.
 """
-function print_bond_table(cryst::Crystal, max_dist; digits=4, atol=1e-12)
+function print_symmetry_table(cryst::Crystal, max_dist; digits=4, atol=1e-12)
     for b in reference_bonds(cryst, max_dist)
```

Comment:
Took me a minute to realize information for a site is retrieved simply through `Bond(j,j,[0,0,0])`. But this is very nice.

Reply:
Thanks for catching this. I have updated the doc string and removed the `print_bond(cryst, site)` function with the new `print_site(cryst, site)` function. It is still true, however, that `Bond(j,j,[0,0,0])` is another way of specifying a site.